iosApp/iosApp/Pages/Search/Results/StopResultsView.swift

```diff
@@ -10,13 +10,13 @@ import shared
 import SwiftUI
 
 struct StopResultsView: View {
-    let stops: [StopResult]
+    let stops: [SearchResultsContainer.Result]
     let handleStopTap: (String) -> Void
 
     var body: some View {
         ResultContainer {
             VStack(spacing: .zero) {
-                ForEach(stops, id: \.id) { stop in
+                ForEach(stops) { stop in
```

Comment:
Why get rid of the ID?

Reply:
It's not needed if the struct conforms to the Identifiable protocol, it'll automatically assign it the id of the element.